src/app.module.ts

```diff
@@ -1,13 +1,28 @@
 
 import {NgModule} from '@angular/core';
 import {BrowserModule} from '@angular/platform-browser';
+import {RouterModule, Routes} from '@angular/router';
+import {StoreModule} from '@ngrx/store';
 
 import {AppComponent} from './app.component';
+import {HelloWorldComponent} from './hello-world/hello-world.component';
```

Comment:
Is this a temporary state? why would we want the app to have both the Todos feature and the hello-world?

I'm fine with this becoming the Todo app (or the Tour of Heroes app) in place of what's currently there, if we still show at least the same features.

Reply:
Todo app was the simplest thing I could think of to demonstrate that ngrx works. Not sure where to take this in the future. Tour of Heroes work be a good demonstration. Or, just have various examples for different Bazel built features on different routes.